cmd/generic-handlers.go

```diff
@@ -139,24 +139,23 @@ func setBrowserCacheControlHandler(h http.Handler) http.Handler {
 	return cacheControlHandler{h}
 }
 
+var validFaviconRegex = regexp.MustCompile(`([^/]+\.js|favicon.ico)`)
```

Comment:
It is needed to add "Cache-Control" for .js and `favicon.ico` , we shouldn't do that for  `.html` ..